### PR TITLE
fixed deprecated normalization in spatial.py

### PR DESCRIPTION
**Pull request by JakeHeffley**
Using normalization within sklearn.linear_model.LassoLars() is deprecated and prints many warnings which slow down refitting for the cnmf. I followed sklearn's suggestion to use a pipeline with StandardScaler instead.

# Description
caiman used a deprecated form of normalization within sklearn.linear_model.LassoLars(). The deprecation warnings print in the command line and slow down computation or freeze the IDE (spyder). 
To fix this I followed sklearn's suggestion to use StandardScalar within a pipeline to normalize the data and set normalization=False within LassoLars(). 

Fixes # (issue)

- [ *] Bug fix (non-breaking change which fixes an issue)

# Has your PR been tested?
Yes, test and demotest

### CODE REVIEW

**Comment by pgunn** [comment]
Thanks for the patch; this looks good.
